### PR TITLE
meetup docs: Add the pages of 1023meetup

### PR DESCRIPTION
**Pull request by ThreeMonth03**
In this pull request, I add the meetup agenda of 10/23.

### CODE REVIEW

File: content/pages/meetup/2024/1023-nycu.rst [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+|                 |                          | 4. Make a simple PR for modmesh                       |    sharing by Wei-Syuan Teng.                |
++-----------------+                          +                                                       +----------------------------------------------+
+| 20:00 ~ 21:00   |                          |                                                       | nsd homework assistance                      |
+|                 |                          |                                                       |                                              |
++-----------------+--------------------------+-------------------------------------------------------+----------------------------------------------+
+| 21:00 ~ 21:30   | free chat                                                                                                                       |                            
++-----------------+--------------------------+-------------------------------------------------------+----------------------------------------------+
```
Comment (ThreeMonth03) [inline content/pages/meetup/2024/1023-nycu.rst:38]: According to @yungyuc's suggestion in pr #482, I schedule the presetation time in the table.

File: content/pages/meetup/2024/1023-nycu.rst [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+This session is for the students who are taking the 
+`numerical software development course <https://yyc.solvcon.net/en/latest/nsd/index.html>`__.
+There are two presentations for the term project.
+
+1. term project (PolygonDust) progress sharing By Han-Xuan Huang.
+2. term project (pybezier) progress sharing by Wei-Syuan Teng.
+
+By the way, if you have any questions about the homework or the project,
+you can come to the meetup and discuss the questions with the teaching assistant.
```
Comment (ThreeMonth03) [inline content/pages/meetup/2024/1023-nycu.rst:95]: I add the detail of the nsd presentation.

**Review by ThreeMonth03 — COMMENTED** [review]
@YenPeiChen07 @ntut-xuan @jteng2127  Could you please review this pull request?

**Comment by jteng2127** [comment]
The white and gray rows are not aligned correctly.
Maybe adding a row in Topic 2 that says "Same as above" would be better?
image

File: content/pages/meetup/2024/1023-nycu.rst [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+* Time: 18:30 -- 21:30 (3 hours)
+
+|
+
+Agenda
+--------
+
+There will be many different subjects discussed at the same time.
+
++-----------------+--------------------------+-------------------------------------------------------+----------------------------------------------+
```
Comment (yungyuc) [inline content/pages/meetup/2024/1023-nycu.rst:22]: The table style may use some improvement (now or in the future):

image

It does not look very clear when the items happen.

File: content/pages/meetup/2024/1023-nycu.rst [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+|                 |                          |                                                       |                                              |
+|                 |                          |                                                       |                                              |
++-----------------+--------------------------+-------------------------------------------------------+----------------------------------------------+
+| 21:00 ~ 21:30   | free chat                                                                                                                       |
+|                 |                                                                                                                                 |
+|                 |                                                                                                                                 |
++-----------------+--------------------------+-------------------------------------------------------+----------------------------------------------+
```
Comment (ThreeMonth03) [inline content/pages/meetup/2024/1023-nycu.rst:44]: Modify the layout.

**Review by ThreeMonth03 — COMMENTED** [review]
@yungyuc @jteng2127 I adjusted the layout few minutes ago. How do you think about current layout of the agenda?

**Comment by yungyuc** [comment]
> @yungyuc @jteng2127 I adjusted the layout few minutes ago. How do you think about current layout of the agenda?

It looks better now.

image

However, I think we need to improve both the typesetting and style in the future.

**Comment by ntut-xuan** [comment]
> > @yungyuc @jteng2127 I adjusted the layout few minutes ago. How do you think about current layout of the agenda?
> 
> It looks better now.
> 
> image
> However, I think we need to improve both the typesetting and style in the future.

Hi @yungyuc. It look better! I'll be happy to support the layout design in future (if time allow).

**Comment by yungyuc** [comment]
> Hi @yungyuc. It look better! I'll be happy to support the layout design in future (if time allow).

Just make a PR.

**Comment by ThreeMonth03** [comment]
Thanks, @jteng2127 @yungyuc and @ntut-xuan. I decide to merge this pull request now. @ntut-xuan, you could make a follow-up pull request if you want to improve the layout.

**Pull request merged**